pkg/util/redirect.go

```diff
@@ -0,0 +1,36 @@
+package util
```

Comment:
This file shouldn't be used anymore and should be removed.

Reply:
OK.